packages/slice-machine/server/src/api/screenshots/generate.ts

```diff
@@ -21,53 +21,28 @@ export interface ScreenshotResults {
   failure: FailedScreenshot[];
 }
 
-export async function generateScreenshotAndRemoveCustom(
-  env: BackendEnvironment,
-  libraryName: string,
-  sliceName: string
-): Promise<ScreenshotResults> {
-  const { screenshots, failure } = await generateScreenshot(
-    env,
-    libraryName,
-    sliceName
-  );
-
-  // Remove custom screenshot of success
-  Object.keys(screenshots).forEach((variationId) =>
-    removeCustomScreenshot(env, libraryName, sliceName, variationId)
-  );
-
-  return {
-    screenshots: screenshots,
-    failure: failure,
-  };
-}
```

Comment:
There are two types of screenshots
- regular -> taken using pupeteer rendering your slice in the context of your project with mock data
- custom -> custom image uploaded by the user.

Slice Machine will look in that order `custom > regular` when searching for a screenshot.

ℹ️  So
- If a user takes a regular screenshot, the custom one must be deleted in order for the regular to show up properly
- If a user adds a custom screenshot it will be taken by default so it's ok.